The raw paired-end Illumina NovaSeq 6000 reads for Chelonia mydas normal (CH-NORMS1) and abnormal (Ab-NormS2) samples were obtained from Macrogen Singapore using the TruSeq Nano DNA kit (151 bp, paired-end).
Quality evaluation with FastQC (v0.11.9) and MultiQC (v1.15) revealed no poor-quality reads, consistent base quality across all positions, and GC content near 44 %.
These results indicate high-quality DNA sequencing suitable for downstream mapping and variant analysis.


# Sequencing Data Summary

| Dataset | File | Encoding | Total Sequences | Total Bases | Read length | %GC | Poor-quality reads |
|---|---|---|---:|---:|---:|---:|---:|
| Normal (CH-NORMS1) | W1-A_1.fastq.gz | Illumina 1.9 | 456,923,725 | 68.9 Gbp | 151 bp | 44 | 0 |
| Normal (CH-NORMS1) | W1-A_2.fastq.gz | Illumina 1.9 | 456,923,725 | 68.9 Gbp | 151 bp | 45 | 0 |
| Abnormal (Ab-NormS2) | S1_1.fastq.gz | Illumina 1.9 | 441,497,634 | 66.6 Gbp | 151 bp | 44 | 0 |
| Abnormal (Ab-NormS2) | S1_2.fastq.gz | Illumina 1.9 | 441,497,634 | 66.6 Gbp | 151 bp | 44 | 0 |
